components/template-settings-json.tpl

```diff
@@ -0,0 +1,9 @@
+{
+  "site": {
+    "product_settings": {
+      "key": "PREFIX_product",
```

Comment:
It's not good name for auto-generated product page BG. It should be reserved for design dependent auto-gen product page settings.

For product page bg key is suggest: `product.product_bg` or `product.bg` and key name (must be uniq in `site`  scope) `PREFIX_product_body_bg` (there has also header bg `PREFIX_product_header_bg`).

```suggestion
  "product": {
    "product_bg": {
      "key": "PREFIX_product_body_bg",
```